### PR TITLE
Fix use of exceptions

### PR DESCRIPTION
**Pull request by JessickaMoya**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java [hunk 1]
```diff
@@ -28,7 +28,6 @@
 import org.springframework.web.bind.annotation.RestController;
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java:28]: Jessi, this class has two copyright could you checked please?

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java [hunk 2]
```diff
@@ -50,27 +50,27 @@ public void convertVideo() throws ConverterException, IOException {
...
         System.out.println(theCommand);
         ThreadHandler errorHandler = new ThreadHandler(process.getErrorStream(), "Error Stream");
         errorHandler.start();
         ThreadHandler threadHandler = new ThreadHandler(process.getInputStream(), "Output Stream");
         threadHandler.start();
         LOGGER.info("start");
-        try {
             LOGGER.info("Execute Try");
             process.waitFor();
             LOGGER.info("finish");
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/core/VideoConverter.java]: I think these 3 loggers are no longer needed: Start, Execute try and finish

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java [hunk 3]
```diff
@@ -100,24 +100,24 @@ private String changeResolution() {
...
         System.out.println(thumbnailCommand);
+        try {
         Process process = Runtime.getRuntime().exec("cmd /c " + thumbnailCommand);
         ThreadHandler errorHandler = new ThreadHandler(process.getErrorStream(), "Error Stream");
         errorHandler.start();
         ThreadHandler threadHandler = new ThreadHandler(process.getInputStream(), "Output Stream");
         threadHandler.start();
         LOGGER.info("start");
-        try {
             LOGGER.info("Execute Try");
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/core/VideoConverter.java:110]: I think these 3 loggers are no longer needed: Start, Execute try and finish

File: src/main/java/org/fundacion/jala/converter/core/facade/ChecksumFacade.java [hunk 4]
```diff
@@ -37,35 +37,36 @@ private ChecksumFacade() {
...
+            List<String> resultPath = getPath(checksum, assets);
+            exist = resultTitle.size() > 0;
+            if (exist) {
+                filename = resultTitle.get(0);
+                storagePath = resultPath.get(0) + filename;
+            } else {
+                filename = file.getOriginalFilename();
+
+                    storagePath = fileStorageService.uploadFile(file);
+
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/core/facade/ChecksumFacade.java]: These empty lines should be removed

File: src/main/java/org/fundacion/jala/converter/core/facade/ExtractFacade.java [hunk 5]
```diff
@@ -36,17 +36,20 @@ private ExtractFacade() {
      *
      * @param file file is image file to extract text.
      * @param  language is a type of language of the text.
-     * @throws IOException is a exception when invalid input is provided.
      * @throws TextExtractorException if process is interrupted.
      */
-    public static void getTextExtract(final MultipartFile file, final String language) throws IOException, TextExtractorException {
+    public static void getTextExtract(final MultipartFile file, final String language) throws TextExtractorException {
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/core/facade/ExtractFacade.java]: This empty line should be removed

**Review by JorgeRCV — APPROVED** [review]

**Review by GustavoZHA — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java (hunk 2 above)
Comment (JessickaMoya) [reply src/main/java/org/fundacion/jala/converter/core/VideoConverter.java]: You are right. I just fixed it

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java (hunk 3 above)
Comment (JessickaMoya) [reply src/main/java/org/fundacion/jala/converter/core/VideoConverter.java:110]: You are right. I just fixed it

File: src/main/java/org/fundacion/jala/converter/core/facade/ChecksumFacade.java (hunk 4 above)
Comment (JessickaMoya) [reply src/main/java/org/fundacion/jala/converter/core/facade/ChecksumFacade.java]: I just deleted it

File: src/main/java/org/fundacion/jala/converter/core/facade/ExtractFacade.java (hunk 5 above)
Comment (JessickaMoya) [reply src/main/java/org/fundacion/jala/converter/core/facade/ExtractFacade.java]: I just deleted it

**Review by jpgonzales27 — APPROVED** [review]

**Pull request merged**